workflows_core/workflow/context_manager.py

```diff
@@ -67,6 +67,9 @@ def __exit__(self, exc_type: type, exc_value: BaseException, traceback: Tracebac
         if self._workflow_id is not None:
             if exc_type is not None:
                 # Handle the except, let user know etc...
+                # print the error (ideally logged but this is good enough for now)
```

Comment:
nah can we got logging from the get go please, avoid double handling

Reply:
Yeah good call  - I'll change it to `logging.debug`